### DISCUSSION TITLE
Setter support?

### DISCUSSION THREAD

**Discussion by minimul** [hopsoft/turbo_boost-streams#30] [Q&A] [answered]
e.g. `.invoke("checked=", args: [true], selector: '#order_form_any_time')`

**Comment by hopsoft** [comment, marked as answer]
Support for this exists. SEE: https://smallteam.tech/@turbo-boost/streams/demos/assignment 

Are you having issues with it?

**Reply by minimul** [reply]
That link is not working.

**Reply by minimul** [reply]
OK, I see the support here => https://github.com/hopsoft/turbo_boost-streams/blob/fe3d244c17f95a71cfd26e14fc7d8299ac2a8f50/app/javascript/invoke.js#L13

So it'll be just as I wrote it out in the question. I'll give it a whirl.

**Reply by hopsoft** [reply]
Yes. It should work as you wrote it above. Let me know how it goes.

**Reply by minimul** [reply]
Works like a charm! thanks @hopsoft
